pkg/registry/scheduling/priorityclass/strategy.go

```diff
@@ -45,12 +46,15 @@ func (priorityClassStrategy) NamespaceScoped() bool {
 func (priorityClassStrategy) PrepareForCreate(ctx context.Context, obj runtime.Object) {
 	pc := obj.(*scheduling.PriorityClass)
 	pc.Generation = 1
+	schedulingutil.DropDisabledFields(pc, nil)
 }
 
 // PrepareForUpdate clears fields that are not allowed to be set by end users on update.
 func (priorityClassStrategy) PrepareForUpdate(ctx context.Context, obj, old runtime.Object) {
-	_ = obj.(*scheduling.PriorityClass)
-	_ = old.(*scheduling.PriorityClass)
+	newClass := obj.(*scheduling.PriorityClass)
+	oldClass := old.(*scheduling.PriorityClass)
+
+	schedulingutil.DropDisabledFields(oldClass, newClass)
```

Comment:
args are reversed here, should be DropDisabledFields(newClass, oldClass)

Reply:
Done. Thank you